react-ui/src/project/index.js

```diff
@@ -23,6 +23,8 @@ const Project = () => {
     const [isCycleTime, setIsCycleTime] = useState(false);
     const [isDevFocus, setIsDevFocus] = useState(false);
     const [isCruft, setIsCruft] = useState(false);
+    const [startDate, setStartDate] = useState('');
+    const [endDate, setEndDate] = useState('');
```

Comment:
You need to pass the start date and end date in the request. Check where **axios** call is being made and see how project slug is being passed.